src/server/game/Entities/Player/Player.cpp

```diff
@@ -2626,6 +2626,7 @@ bool Player::IsGroupVisibleFor(Player const* p) const
         default: return IsInSameGroupWith(p);
         case 1:  return IsInSameRaidWith(p);
         case 2:  return GetTeam() == p->GetTeam();
+        case 3:  return nullptr;
```

Comment:
like I said it compiled and worked fine for me, tested it before I opened this PR, but now it's changed and it still works.